spec.md

```diff
@@ -1014,13 +1020,14 @@ For success responses, the following fields are defined:
 
 ## Binding
 
-If `bindable:true` is declared for a service or plan in the
+If `"bindable": true` is declared for a service or plan in the
 [Catalog](#catalog-management) endpoint, the Platform MAY request generation
-of a Service Binding.
+of a Service Binding. Otherwise, Platforms MUST NOT make a binding request to
+the Service Broker any Service Instance using the given service or plan.
```

Comment:
for any Service Instance

Reply:
@mattmcneeney did you want to update this?

Reply:
Thanks, this one slipped past me. Updated!